_posts/2017-04-12-launching-statesman-votetracker.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: post
+title: "Lessons learned working on Statesman VoteTracker"
+date: 2017-04-11 10:00:00 -0500
+categories: votetracker
+byline:
+    - name: "Christian McDonald"
+      twitter: "crit"
+---
+
+![VoteTracker](/assets/img/2017-04-12-votetracker-top.png)
+
+Last week, after a 3-year journey, the Statesman news apps team launched [VoteTracker](http://apps.statesman.com/votetracker/entities/austin-city-council/), a look into the voting history of Austin City Council members on some of the more important issues that cross the dias.
+
+Then city hall reporter [Lilly Rockwell](https://twitter.com/LillyRockwell) first approached the news apps team some time in 2014 as the first 10-1 council was up for election. There was no "data" on agenda items or votes then, and it isn't much different today. The city now posts [agenda items on their data portal](https://data.austintexas.gov/Government/Austin-City-Council-Agenda-Items/es7e-878h/data), but the descriptions are still obtuse and there isn't any follow-up data on voting. The only vote record is posted in a PDF word doc in minutes approved at the following meeting.
+
+In the three years since, we worked on VoteTracker more off than on. We created a "minimum viable product" admin and then abandoned it. We built and API back end with plans to develop a Javascript framework frontend, and then abandonded it. In the end, we settled on a tried-and-true method for us: A [django](https://www.djangoproject.com/) admin to manage data, and L.A. Times' [django-bakery](https://github.com/datadesk/django-bakery)-based views to publish static pages to Amazon S3. 
+
+
+### Where's the data?
+
+Without a data source of outcomes, building a tracker of all votes at City Hall is a dubious challenge. OK, buidling it is not the problem ... maintaining it is. Even with a data source of agenda items, we don't have the manpower (or interest, frankly) to enhance each item with every discussion, amendment and vote in a typical 100-item agenda.
+
+So, we settled on concentrating on the same newsworthy agenda items we typically write about on the City Hall beat. Hopefully, as we use VoteTracker more, we can use it as an endpoint for our coverage, allowing reporters to spend more time on specific issues that matter to most people, while at the same time offering more coverage of things we haven't been able to spend time, ink or pixels on in the past.
+
+Because of this, VoteTracker is more of a browsing experience than a database of record, and therefore we don't have a "data" to share either. Perhaps some day the city will advance their record keeping and we could use something like [Councilmatic](https://www.councilmatic.org/) in the future.
+
```

Comment:
I feel like "where's the data" isn't answered super clearly because of how the second and third grafs in this section are structured. Maybe like this?

> We decided to concentrate only on the newsworthy agenda items we typically write about on the City Hall beat, making VoteTracker more of a browsing experience than a database of record.
>
> Hopefully, as we use VoteTracker more, we can use it as an endpoint for our coverage, allowing reporters to spend more time on specific issues that matter to most people, while at the same time offering more coverage of things we haven’t been able to spend time, ink or pixels on in the past.